run_patch.py

```diff
@@ -1,6 +1,6 @@
 # bchhun, {2020-02-21}
 
-from pipeline.patch_VAE import extract_patches, build_trajectories
+from pipeline.patch_VAE import extract_patches, build_trajectories, generate_trajectory_relations
```

Comment:
`generate_trajectory_relations` is now merged to the step `assemble_VAE`, where all data from the same well are processed together.